modules/dxp/apps/portal-search-tuning/portal-search-tuning-gsearch-configuration-web/src/main/resources/META-INF/resources/js/components/Fragment.js

```diff
@@ -154,11 +156,12 @@ export default function Fragment({
 }
 
 Fragment.propTypes = {
-	collapse: PropTypes.number,
+	collapseAll: PropTypes.bool,
 	deleteFragment: PropTypes.func,
 	description: PropTypes.string,
 	disabled: PropTypes.bool,
 	icon: PropTypes.string,
-	json: PropTypes.string,
+	jsonString: PropTypes.string,
 	title: PropTypes.object,
+	updateJson: PropTypes.func,
```

Comment:
Can you remove the `updateJson` prop?

Reply:
Oh why? I thought we needed it...

Reply:
Oh whoops, I thought it was leftover code.